src/main/kotlin/org/rust/lang/core/resolve/ref/RsLitExprReferenceContributor.kt

```diff
@@ -36,7 +38,8 @@ class RsLitExprReferenceContributor : PsiReferenceContributor() {
     override fun registerReferenceProviders(registrar: PsiReferenceRegistrar) {
         registrar.registerReferenceProvider(includeMacroLiteral or pathAttrLiteral or pathValueLiteral, RsFileReferenceProvider())
         // LOWER_PRIORITY is used to have ability to override reference if needed
-        registrar.registerReferenceProvider(literal, RsLiteralWebReferenceProvider(), PsiReferenceRegistrar.LOWER_PRIORITY)
+        registrar.registerReferenceProvider(literal, RsLiteralGeneralUrlProvider(), PsiReferenceRegistrar.LOWER_PRIORITY)
+        registrar.registerReferenceProvider(literal, RsLiteralGitHubIssueProvider(), PsiReferenceRegistrar.LOWER_PRIORITY)
```

Comment:
nit: Actually, I think we can drop `PsiReferenceRegistrar.LOWER_PRIORITY` for `RsLiteralGitHubIssueProvider` because it's unlikely something may want to provide another reference for issue number literal